packages/dart_frog_test/lib/src/dart_frog_test.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:async';
+
+import 'package:dart_frog/dart_frog.dart';
+import 'package:meta/meta.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:test/test.dart';
+
+class _MockRequestContext extends Mock implements RequestContext {}
+
+class _MockRequest extends Mock implements Request {}
+
+/// A test that can be used to test a Dart Frog handler.
+@isTest
+void testDartFrog(
```

Comment:
The name doesn't convince me, I feel Dart Frog is way to generic as the subject of the test since we are testing a specific endpoint.

Maybe `testRoute`, `testMiddleware` and so on?

Reply:
yeah, good call, maybe `testRouteHandler`?